#{"labels_list": ["green_go", "pedestrian_crossing", "red_stop", "speed_limited", "speed_minimum", "speed_unlimited", "yellow_back"], "image_shape": ["352", "640"]}
int32[7] flag
int32[7] x_min
int32[7] y_min
int32[7] x_max
int32[7] y_max
float32[7] score